pkg/reconciler/kubernetes/tektonconfig/tektonconfig.go

```diff
@@ -119,6 +121,20 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonConfi
 
 	var stages common.Stages
 	if tc.Spec.Profile == common.ProfileLite {
+		_, err := tt.GetTrigger(r.operatorClientSet.OperatorV1alpha1().TektonTriggers(), common.TriggerResourceName)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				return nil
```

Comment:
here if the error is `NotFound` then we should not do anything, the execution flow down to the `delete` call, shouldn't it.

Reply:
on second thougt, does this mechanism really need to do a `Get` API call. 
could did directly call `Delete` and ignore `NotFound` error if the item doesnot exist.

Reply:
updated !